rust/tools/authorization-logic/README.md

```diff
@@ -77,11 +77,19 @@ claim, and the recipient uses the public key to check the signature.
 To compile a program called FILENAME stored in INPUT_DIR, run:
 
 ```
-cargo run FILENAME INPUT_DIR OUTPUT_DIR
+cargo run -- --in-dir INPUT_DIR --out-dir OUTPUT_DIR FILENAME
```

Comment:
Is the first `--` just because this is being run through cargo? If we build the binary, and run it outside of cargo, would we drop the first `--`? Just curious.